### ISSUE TITLE
Get-HPOVEnclosure throws error when using -Report 

### ISSUE THREAD

**Issue by salcinad** [HewlettPackard/POSH-HPEOneView#389]
### Expected Behavior
Get a full report of the Synergy 12000 Frame .

### Actual Behavior
get following error and cmdlets fails
```
      Interconnect Name            Logical Interconnect     State      Status Consistency state
----- -----------------            --------------------     -----      ------ -----------------
      Frame01, interconnect 1 LE_1-LIG_SAS-1      Configured OK                      
      Frame01, interconnect 3 LE_1-LIG_VC_Default Configured OK     Consistent       
      Frame01, interconnect 4 LE_1-LIG_SAS-1      Configured OK                      
      Frame01, interconnect 6 LE_1-LIG_VC_Default Configured OK     Consistent       
Enclosure-Report : Cannot validate argument on parameter 'uri'. -URI must being with a '/' (eg. /rest/server-hardware) in its value. Please correct the value and try again.
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.410\4.10.1889.2173\HPOneView.410.psm1:41425 char:45
+ ...        $_EnclosureCollection | ForEach-Object { Enclosure-Report $_ }
+                                                     ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Enclosure-Report], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Enclosure-Report
```

### Steps to reproduce
```
Connect-HPOVMgmt -Hostname $IP -UserName $username -Password $password -AuthLoginDomain "local"
$Enclosure = Get-HPOVEnclosure -Name "*"
Get-HPOVEnclosure -Name ($Enclosure).Name -report
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`): 
`4.10.1889.2173 C:\Program Files\WindowsPowerShell\Modules\HPOneView.410\4.10.1889.2173`

HPE OneView **Composer** Version (`Get-HPOVVersion -ApplianceVer`):   ApplianceVersion: 
`4.10.04.370820.00 4.10.1889.2173 C:\Program Files\WindowsPowerShell\Modules\HPOneView.410\4.10.1889.2173`

Output from `$PSVersionTable` on your Windows Host:
```
Name                           Value                                                                                                                                                                             
----                           -----                                                                                                                                                                             
PSVersion                      5.1.14409.1005                                                                                                                                                                    
PSEdition                      Desktop                                                                                                                                                                           
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                           
BuildVersion                   10.0.14409.1005                                                                                                                                                                   
CLRVersion                     4.0.30319.42000                                                                                                                                                                   
WSManStackVersion              3.0                                                                                                                                                                               
PSRemotingProtocolVersion      2.3                                                                                                                                                                               
SerializationVersion           1.1.0.1
```

**Comment by salcinad** [comment]
Tested with 4.10.1954.1523, still get that error. I could send you output of Get-HPOVCommandTrace if needed, but not over github. I did already send you an email on 01.02.2019 regarding this issue.

Here in log you can see also that i am using new version.
```
Enclosure-Report : Cannot validate argument on parameter 'uri'. -URI must being with a '/' (eg. /rest/server-hardware) in its value. Please correct the value and try again.
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.410\4.10.1954.1523\HPOneView.410.psm1:47078 char:45
+ ...        $_EnclosureCollection | ForEach-Object { Enclosure-Report $_ }
+                                                     ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Enclosure-Report], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Enclosure-Report
```

**Comment by ChrisLynchHPE** [comment]
I found the issue.  It has to do with identifying the associated Logical Interconnect resource, and the property names are different from Ethernet/FC and SAS Logical Interconnects.  I'll have that fixed in the next updates for 4.10 and 4.20 libraries.

**Comment by ChrisLynchHPE** [comment]
This was fixed in prior library releases.  However, here is a current list of released versions:

* [5.50.2680.1687](https://www.powershellgallery.com/packages/HPEOneView.550/5.50.2680.1687) 
* [5.40.2681.1725](https://www.powershellgallery.com/packages/HPEOneView.540/5.40.2681.1725)
* [5.30.2682.3811](https://www.powershellgallery.com/packages/HPEOneView.530/5.30.2682.3811)
* [5.20.2684.2161](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2684.2161)
* [5.00.2685.1865](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2685.1865)

Reminder:  The 4.10 and 4.20 libraries are no longer supported.  You need to update your OneView appliances, and the associated PowerShell libraries.

Closing due to no further activity.  If you feel this is in error, please feel free to re-open this issue.
